infra.bs

```diff
@@ -119,6 +119,27 @@ will make the caller automatically rethrow the given value, if any, and thereby
 caller's algorithm. Using prose the caller has the ability to "catch" the exception and perform
 another action.
 
+<h4 id=algorithm-iteration>Iteration</h4>
+
+<p>There's a variety of ways to repeat a set of steps until a condition is reached.
+
+<p class=warning>The Infra Standard is not (yet) exhaustive on this, please file an issue if you
+need something.
+
+<p>For each, as defined <a for=list lt="for each">for lists</a> (and derivatives) and
```

Comment:
Hmm, these look weird as stand-alone paragraphs; I think it'd be best to go back to a bulleted list